api/data/question-indicators.sql

```diff
@@ -0,0 +1,77 @@
+INSERT INTO question_indicator_map ("indicator", "question") VALUES
+    -- Not questions in the survey but widgets only
```

Comment:
I totally forgot why we need this map or how it helps improving something.

I've seen that in survey_answers we have 2 columns that literally references both columns of this table. Why can't we just store the values in the surve_answers table?